integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -144,4 +145,31 @@ context('Create goals', () => {
         ),
     )
   })
+
+  it(`should display 'goals created successfully' message`, () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    let overviewPage = Page.verifyOnPage(OverviewPage)
+    overviewPage.doesNotHaveGoalsCreatedSuccessfullyMessage()
+
+    const createGoalPage = overviewPage.clickAddGoalButton()
+
+    createGoalPage //
+      .setGoalTitle('Learn French', 1)
+      .setTargetCompletionDate6to12Months(1)
+      .setStepTitle('Book course', 1, 1)
+
+    // When
+    createGoalPage.submitPage()
+
+    // Then
+    overviewPage = Page.verifyOnPage(OverviewPage)
+    overviewPage //
+      .hasGoalsCreatedSuccessfullyMessage()
+      .refreshPage()
+      .doesNotHaveGoalsCreatedSuccessfullyMessage()
```

Comment:
Ensure the message is only displayed once, immediately after creating goals.
The starts on the Overview screen and asserts the message is not show. Then it creates a goal, arrives back on the Overview screen and asserts the message is displayed. Then the page is refreshed, and we assert the message is no longer displayed